corehq/apps/app_manager/tasks.py

```diff
@@ -89,12 +89,6 @@ def update_linked_app_and_notify_task(domain, app_id, master_app_id, user_id, em
     update_linked_app_and_notify(domain, app_id, master_app_id, user_id, email)
 
 
-@task
-def load_appcues_template_app(domain, username, app_slug):
-    from corehq.apps.app_manager.views.apps import load_app_from_slug
-    load_app_from_slug(domain, username, app_slug)
```

Comment:
I think you can delete this function altogether. Its only remaining purpose is to give devs a way to load template apps (see [here](https://github.com/dimagi/commcare-hq/blob/master/DEV_FAQ.md#applications)), and I'm skeptical anyone would miss it. That should lead to deleting a bunch more code, including the JSON blobs of the template apps themselves.

Reply:
I'll ask to see if anyone uses it, but I suspect you're right and it'll be fine to remove.

Reply:
Given there's some talk of potentially using template apps for other purposes later (like the option to start from a template instead of a blank app in app builder), I think I prefer to do that removal in a separate PR so it can be more easily reverted if we decide to use that functionality again (without accidentally reintroducing _default_ template apps to new accounts).

I have that [in a branch](https://github.com/dimagi/commcare-hq/tree/em/remove-template-apps-unused-code) now, not PR'd yet because of an open question on whether [this test file](https://github.com/dimagi/commcare-hq/commit/d5e2bb7c3b6de448e2dc5defc2112ad37f7ee30f) that references a template app is okay to delete.